website/docs/tutorials/07-testing/03-fuzzing/strategies.md

```diff
@@ -75,6 +75,17 @@ Fuzzer picks integers from provided range and then converts them to felts.
 If `min_value` is not `None` then all values will be greater than or equal to `min_value`,
 and if `max_value` is not `None` then all values will be less than or equal to `max_value`.
 
+### `strategy.short_strings`
+```python
+def short_strings() -> Strategy:
+```
+
+Generates strings with ascii characters of length that passes the condition `0 <= length <= 31`.
```

Comment:
```suggestion
Generates strings with ASCII characters of length that passes the condition `0 <= length <= 31`.
```

Reply:
[ASCII instead of ascii in docs](https://github.com/software-mansion/protostar/pull/886/commits/5f354b9bf0a1801235d9f031e84ce461f3dc3bfc)